### PR TITLE
https://issues.redhat.com/browse/ACM-11979--adding to what's new, las…

### PR DESCRIPTION
**Pull request by swopebe**
…t clean up of TP and description

### CODE REVIEW

**Review by rokej — APPROVED** [review]

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ When you use hosted control planes for {ocp} on Amazon Web Services (AWS), the i
 [#hosted-aws-infra-iam-prereqs]
 == Prerequisites
 
-* You must configure hosted control planes before you can create hosted control plane clusters. See xref:../../clusters/hosted_control_planes/aws_intro.adoc#hosting-service-cluster-configure-aws[Configuring hosted control plane clusters on AWS (Technology Preview)] for more information.
+* You must configure hosted control planes before you can create hosted control plane clusters. See xref:../../clusters/hosted_control_planes/aws_intro.adoc#hosting-service-cluster-configure-aws[Configuring hosted control plane clusters on AWS] for more information.
```
Comment (dockerymick) [inline clusters/hosted_control_planes/manage_aws_infra_iam.adoc:14]: Alignment for YAML callouts is not rendering
Screenshot 2024-06-28 at 2 06 45 PM

File: clusters/release_notes/whats_new.adoc [hunk 2]
```diff
@@ -42,5 +42,9 @@ Learn about new features and enhancements for Cluster lifecycle with {mce-short}
 [#hosted-control-plane]
 == Hosted control planes
 
+Configuring hosted control plane clusters on AWS is now generally available. You can deploy the HyperShift Operator on an existing managed cluster by using the `hypershift-addon` managed cluster add-on to enable that cluster as a hosting cluster and start to create the hosted cluster. 
+
+See xref:../clusters/hosted_control_planes/aws_intro.adoc[Configuring hosted control plane clusters on AWS] for details.
```
Comment (dockerymick) [inline clusters/release_notes/whats_new.adoc]: broken link
Screenshot 2024-06-28 at 2 08 36 PM
```suggestion
See xref:../../clusters/hosted_control_planes/aws_intro.adoc[Configuring hosted control plane clusters on AWS] for details.
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/hosted_control_planes/manage_aws_infra_iam.adoc:14]: @dockerymick this PR didn't change this, but we can add it to a long list of things that HCP team has to address on the doc. for my issue, it is to remove TP from the title.

**Review by dockerymick — APPROVED** [review]
Thanks for the reminder about tech debt fixes on the way. Links and other changes look great!

**Pull request merged**